beets/ui/commands.py

```diff
@@ -613,6 +613,9 @@ def choose_candidate(candidates, singleton, rec, cur_artist=None,
         })
         if default is None:
             require = True
+        # Bell ring when user interaction is needed.
+        if config['import']['bell']:
+            print('\a')
```

Comment:
This seems to add an extra newline, in addition to ringing the bell. Perhaps you mean `print('\a', end='')`?

Also, all output from the commands needs to go through our custom `ui.print_` function instead of the built-in `print` for dumb encoding reasons. Could you please use that function?